rebar.config

```diff
@@ -1,12 +1,13 @@
 %% == Erlang Compiler ==
 
-{minimum_otp_vsn, "19.3"}.
+{minimum_otp_vsn, "20.3"}.
```

Comment:
Isn't it too soon to bump?

(Which rule are we to apply, OTP version support wise? The last 5 or the last 3 versions? This change matches neither rule.)

Reply:
I have no rules to apply, OTP version support wise. (if you're relating to the company's internal policy, which I'm not sure applies to external projects)

I bumped because `proper` requires that bump. If you prefer I can bump further to OTP 21.3, but I'd rather not unless it's perfectly required.

**Note**: once we decide on the support policy then, yes, we might decide on bumping this further.